createmsi.py

```diff
@@ -215,9 +215,211 @@ def generate_files(self):
             'Value': 'INSTALLDIR',
         })
         if platform.system() == "Windows":
-            ET.SubElement(product, 'UIRef', {
-                'Id': 'WixUI_FeatureTree',
-            })
+            if self.license_file is None:
+                ui = ET.SubElement(product, 'UI', {
```

Comment:
Whenever you add functionality that has this many lines of new code, it needs to be put in its own method rather whan putting it inside an existining method.

Reply:
Done